### PR TITLE
Main repo changes

### PR DESCRIPTION
**Pull request by Sithyphus**
The master branch has a lot of the rgb stuff it was based on so this merge is basically just updating and changing everything for lidar.

### CODE REVIEW

**Comment by Sithyphus** [comment]
I'll be resolving some merge conflicts today for this.

File: get_files_from_drive.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import argparse
```
Comment (Chris-Schnaufer) [inline get_files_from_drive.py:1]: I don't think we should provide this since we'd have to maintain it if Google changes things.

**Review by Chris-Schnaufer — CHANGES_REQUESTED** [review]

File: get_files_from_drive.py (hunk 1 above)
Comment (Sithyphus) [reply get_files_from_drive.py:1]: So what should I do with it? It's kinda needed for the travis.yml for testing right? Should I just copy the code into the yml?

Comment (Chris-Schnaufer) [reply get_files_from_drive.py:1]: You are correct. nvmd

**Review by Chris-Schnaufer — APPROVED** [review]

File: algorithm_lidar.py [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+"""Plot-level Lidar algorithm
 """
 
 # Importing modules. Please add any additional import statements below
 import numpy as np
 
 # Definitions
 # Please replace these definition's values with the correct ones
-VERSION = '1.0'
+VERSION = '0.0.1'
```
Comment (Chris-Schnaufer) [inline algorithm_lidar.py]:
```suggestion
VERSION = '1.0'
```

**Review by Chris-Schnaufer — APPROVED** [review]

**Pull request merged**